js/packages/teams-ai/src/TaskModules.ts

```diff
@@ -159,6 +159,10 @@ export class TaskModules<TState extends TurnState> {
 
                     // Call handler and then check to see if an invoke response has already been added
                     const result = await handler(context, state, context.activity.value?.data ?? {});
+                    
+                    // if result is null or undefined, this indicates no response should be sent 
+                    if (!result) return;
```

Comment:
This is incorrect handling. Because this is an invoke, it is expected that a response is sent back, even if there is no body to the response. Like your example code showed, instead of `return`, it should send the following: 

```typescript
 await context.sendActivity({
          value: { status: 200 } as InvokeResponse,
          type: ActivityTypes.InvokeResponse
         });
```